File: TheAnvil/src/main/java/com/qualityplus/anvil/base/requirement/VanillaEnchantRequirement.java
package com.qualityplus.anvil.base.requirement;

import com.qualityplus.assistant.lib.eu.okaeri.configs.OkaeriConfig;
import lombok.*;

import java.util.Map;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode(callSuper = true)
public class VanillaEnchantRequirement extends OkaeriConfig {
    private Map<Integer, Integer> requiredLevelsToEnchant;
}

File: TheAnvil/src/main/java/com/qualityplus/anvil/base/requirement/VanillaEnchantRequirement.kt
package com.qualityplus.anvil.base.requirement

import eu.okaeri.configs.OkaeriConfig
import lombok.*

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode(callSuper = true)
class VanillaEnchantRequirement : OkaeriConfig() {
    private val requiredLevelsToEnchant: Map<Int, Int>? = null
}